### DISCUSSION TITLE
Correct pixels white horizontally

### DISCUSSION THREAD

**Discussion by b1zar** [ImageMagick/ImageMagick#2419] [Help]
Hey. Is it possible to align pixels of the same color horizontally using ImageMagic tools without losing the structure?

There are images like this:
https://ibb.co/f13XT2J

So that only letters remain, I use the command:
magick Captcha1.png -sharpen 7x7 Captcha2.png
magick Captcha2.png -color-threshold "sRGB (0,0,0) -sRGB (50,50,50)" monet.png

It turns out this result:
https://ibb.co/LzB0QCF

Due to the waves, the OCR module cannot understand the sequence of letters and give a more accurate result. 
I manually aligned the white pixels horizontally through the editor and the result is better.

What I've done: https://ibb.co/dGdWb95

As a result, the module with the final image became easier to train than in the original version.

Is it possible to achieve horizontal alignment of pixels using Imagemagic?

version: ImageMagick-7.0.10-24-Q16-HDRI-x64-dll, Windows 10

**Comment by dlemstra** [comment]
This looks like you are trying to break a CAPTCHA. We won't help you with that.
